WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -206,8 +417,80 @@ - (NSString *)cellIdentifierForIndexPath:(NSIndexPath *)indexPath
             }
             break;
 
-        case StatsSectionGraph:
         case StatsSectionPeriodHeader:
+            return StatsTablePeriodHeaderCellIdentifier;
+
+        case StatsSectionTagsCategories:
+        case StatsSectionPublicize:
+        {
```

Comment:
Maybe overkill, but would it make sense to define consts for the row numbers in this method?